### ISSUE TITLE
Why is RwCursor not implemented in rkv crate?

### ISSUE THREAD

**Issue by kkonevets** [mozilla/rkv#175]
I have found RoCursor, but no RwCursor, any reason?

**Comment by ncloudioj** [comment]
I can't recall any particular reason for not exposing the RwCursor. I'd assume that we just haven't found a use case for it yet, so it could be implemented in the following versions if needed.

**Comment by kkonevets** [comment]
the use case is like so: need to bulk insert sorted list

**Comment by ncloudioj** [comment]
Agreed, that's a reasonable feature request. We will look into it for sure.
